### ISSUE TITLE
Docs: Typo in the storefront api for deleting customer address

### ISSUE THREAD

**Issue by ymaheshwari1** [medusajs/medusa#1942]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/medusajs/medusa/issues

### Issue Summary

There is a type in the line `Removes an Address from the Customer's saved addresse.`

### How can this issue be resolved?

1. Fix the type in word `addresse`

### Are you interested in working on this issue?

- [X] I would like to fix this issue

**Comment by shahednasser** [comment]
Hi @ymaheshwari1 thank you for reporting this. We currently have the PR #1883 that fixes up many issues in the API reference including some typos, so this should be fixed once the PR is merged 🙂

**Comment by ymaheshwari1** [comment]
@shahednasser The typo is not fixed in the above mentioned PR: https://github.com/medusajs/medusa/pull/1883/files#diff-60e1b66b0c31a621d0d2ae6be1dc18d05b1cd0a4f32da83125da0e36d5b14c3cR4

**Comment by shahednasser** [comment]
You can see the fix [here](https://github.com/medusajs/medusa/blob/1d822a43d36734ec594b3db9d83226c8c3c8fb9b/packages/medusa/src/api/routes/store/customers/delete-address.ts).
